core/tls-websocket-server/src/connection.rs

```diff
@@ -123,23 +127,37 @@ where
 	/// Returns a boolean 'connection should be closed'.
 	fn read_or_initialize_websocket(&mut self) -> WebSocketResult<bool> {
 		if let StreamState::EstablishedWebsocket(web_socket) = &mut self.stream_state {
+			debug!(
```

Comment:
If we merge this into master, I believe this will be a bit of a spamming with all these incoming `debug` messages here. How about changing them to `trace`?

Reply:
There shouldn't be any log spamming, because we don't log `debug`?

Reply:
If you enable `debug`, it will spam, though. But if you think they should stay on the level of debug, that's fine.

Reply:
IMO enabling `debug` on all the modules will always create a huge log output (not only from this module). If you want `debug` logs without 'spam', you should enable it selectively.